app/assets/stylesheets/components/_highlight-boxes.scss

```diff
@@ -14,9 +14,13 @@
   list-style-type: none;
   padding: 5px 10px 25px 0;
   width: 100%;
-  min-width: 250px;
 
   @include media(tablet) {
+    width: 50%;
+    padding: 5px 25px 25px 0;
+  }
+
+  @include media(desktop) {
     width: 33%;
```

Comment:
Slightly outside the scope of this PR but just noticed this doesn't make full use of the width available. Could do either 33.3333% or try using calc() (if you do use calc, worth checking in older browsers).